spec/api-net-spec.ts

```diff
@@ -623,6 +623,36 @@ describe('net module', () => {
         }, { dest });
       }
 
+      for (const [priorityName, urgency] of Object.entries({
+        throttled: 'u=5',
+        idle: 'u=4',
+        lowest: '',
+        low: 'u=2',
+        medium: 'u=1',
+        highest: 'u=0'
+      })) {
+        for (const priorityIncremental of [true, false]) {
+          test(`should set priority to ${priorityName}/${priorityIncremental} if requested`, async () => {
+            const urlRequest = net.request({
+              url: 'https://httpbin.org/get',
```

Comment:
Access to remote origins is a no-go for test suites, please stand up a server for this test (plenty of examples to go on)

Reply:
The net stack doesn't send the Priority header over http 1 connections. Does the test framework support http2?

Reply:
the "test framework" is node.js: https://nodejs.org/docs/latest/api/http2.html#server-side-example

Reply:
`http2.createServer()`

Reply:
I just have checked the net helpers, they're http 1 only.  Should I extend the respondNTimes to support http 2 or just modify the priority test to use http2.server directly? The former request much more changes. I'd like to hear your opinions.

Reply:
never mind, I have figured it out.

Reply:
the test works in main process now, but it doesn't work in utility process since the session module isn't available, any workaround for this
```javascript
      session.defaultSession.setCertificateVerifyProc((req, cb) => cb(0));
      defer(() => {
        session.defaultSession.setCertificateVerifyProc(null);
      });
```

or it's ok to move the test out and make it run in main process only?